src/pdf/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -21,12 +21,15 @@ export interface ShippingLabelData {
     delivery_instructions?: string;
 }
 
-const mmToPixels = (mm: number): number => {
-    const pixelsPerMmAt72Dpi = 72 / 25.4;
-    return mm * pixelsPerMmAt72Dpi;
-};
+// This is a hard requirement because of the foodbank's printer settings, note that height is adjustable but preferably set at 62mm
```

Comment:
Looks like the change talked about wasn't committed. The height is a hard requirement, fixed at 62mm, the width is technically adjustable but preferred at 150